src/sinks/aws_s_s/sqs/mod.rs

```diff
@@ -0,0 +1,12 @@
+mod config;
+mod integration_tests;
+
+use super::{
+    client::Client,
+    config::{message_deduplication_id, message_group_id, BaseSSSinkConfig},
+    request_builder::{SSRequestBuilder, SendMessageEntry},
+    service::SendMessageResponse,
+    sink::SSSink,
+};
+
+mod client;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏can go up with the other modules